helper-cli/src/main/kotlin/commands/repoconfig/GenerateScopeExcludesCommand.kt

```diff
@@ -71,7 +71,7 @@ internal class GenerateScopeExcludesCommand : CliktCommand(
 
 private fun OrtResult.generateScopeExcludes(): List<ScopeExclude> {
     val projectScopes = getProjects().flatMap { project ->
-        dependencyNavigator().scopeNames(project)
+        dependencyNavigator.scopeNames(project)
```

Comment:
> While at it, make the dependency navigator in OrtResult a property; this simplifies access from clients.

Is there a reason to not do it like this from the beginning, when the dependency navigator in OrtResult was introduced?

Reply:
Well, I have to admit, that this course is not really straight-forward. Initially, I thought there might be more dynamic required to select the navigator implementation. Do you want me to change the commits?

Reply:
It would certainly be nice to get this part of this change squashed into the commit that introduces the dependency navigator. Hoping that you don't run into too many conflicts.

Reply:
Or at least the change to a property should be extracted to a commit proceeding this commit directly.